opensrp-anc/build.gradle

```diff
@@ -178,7 +178,7 @@ tasks.withType(Test) {
 
 dependencies {
     implementation 'androidx.appcompat:appcompat:1.3.1'
-    implementation('org.smartregister:opensrp-client-native-form:2.1.17-SNAPSHOT@aar') {
+    implementation('org.smartregister:opensrp-client-native-form:3.0.0-PREVIEW-SNAPSHOT@aar') {
```

Comment:
Why are we using a PREVIEW release here?  Why has the work not been merged to master?

Reply:
Preview is used here because the latest translation changes are not merged into master yet